cmd/api/controller/controller.go

```diff
@@ -17,18 +42,39 @@ func NewPessoaController(service pessoa.Service) *PessoaController {
 }
 
 func (c *PessoaController) Create(ctx *fiber.Ctx) error {
-	var pessoa *domain.Pessoa
+	pessoareq := PessoaRequest{}
```

Comment:
Lembra que eu comentei que tem um jeito que eu gosto de fazer para criar esses DTOs que é mais minimalista?

É assim que eu costumo fazer, com structs anonimos:

```suggestion
	var pessoareq struct{
		Name       string   `json:"nome" validate:"required,max=32"`
		CpfCnpj    string   `json:"cpfcnpj" validate:"required,max=14"`
		Nascimento string   `json:"nascimento" validate:"required,datetime=2006-01-02"`
		Seguros    []string `json:"seguros" validate:"dive,max=32"`
	}
```

As vantagens são:

1. Não preciso de inventar um nome comprido para cada DTO de cada rota.
2. A definição do struct fica mais próxima do local onde é usado, o que torna isso mais legível
3. Do ponto de vista do leitor fica obvio que esse struct só é usado dentro dessa função e em nenhum outro lugar, o que ajuda bastante a reduzir a complexidade cognitiva da coisa.

Reply:
Feito!